app/assets/images/payment_icons/sam.svg

```diff
@@ -0,0 +1 @@
radientTransform="translate(0 78)"><stop offset="0" stop-color="#ec1d2f"/><stop offset="1" stop-color="#c3202f"/></linearGradient><path d="M11.929 13.516c.259.12.479.519.479.897v4.707c0 .379-.219.598-.479.479l-3.63-1.595c-.259-.12-.479-.519-.479-.897V12.4c0-.379.219-.598.479-.479l3.63 1.595z" fill="url(#pi-SVGID_4_)"/><g><path d="M16.855 13.795l-.04.16a1.32 1.32 0 00-.02.479c.02.12.08.219.14.299.08.08.16.14.259.16.1.04.219.04.319.04.279 0 .519-.06.738-.199.219-.14.359-.359.419-.698a.662.662 0 00-.02-.399.656.656 0 00-.219-.299c-.1-.1-.219-.179-.339-.259-.14-.08-.259-.16-.399-.239-.14-.08-.279-.16-.399-.259-.12-.1-.239-.199-.319-.319-.1-.12-.16-.259-.199-.419-.04-.16-.04-.339 0-.538.06-.239.14-.459.259-.638.12-.179.259-.339.439-.479.179-.12.359-.219.578-.279a2.5 2.5 0 01.658-.1c.499 0 .858.12 1.077.379.219.259.259.658.12 1.217h-.738c.04-.16.04-.299.04-.419s-.02-.219-.06-.319-.12-.16-.219-.199c-.1-.04-.239-.08-.399-.08-.1 0-.219.02-.319.04s-.199.08-.299.14c-.1.06-.179.14-.239.259-.08.1-.12.239-.16.379-.04.199-.02.379.08.499.1.14.219.259.379.359.16.1.339.219.519.319.179.1.359.219.519.359s.279.299.359.499c.08.199.1.439.04.738-.06.299-.16.538-.299.738-.14.199-.299.359-.479.479s-.379.199-.598.259a2.47 2.47 0 01-.658.08c-.279 0-.499-.02-.658-.08a.982.982 0 01-.399-.239.832.832 0 01-.199-.339 1.38 1.38 0 01-.06-.399c0-.12 0-.259.04-.379.02-.12.04-.219.06-.319h.698v.015zM22.858 9.926h.897l.638 5.504h-.778l-.12-1.316h-2.134l-.678 1.316h-.718l2.893-5.504zm.299.658l-1.496 2.932h1.755l-.259-2.932zM26.048 15.43h-.658l1.137-5.504h1.137l.419 4.627h.02l2.373-4.627h1.157l-1.137 5.504h-.698l1.017-4.886h-.02l-2.553 4.886h-.678l-.499-4.886h-.02l-.997 4.886z" fill="#3f494f"/><g fill="#3f494f"><path d="M32.789 14.333l-.04.179h-.259l-.199.917h-.239l.199-.917h-.259l.04-.179h.757zM32.669 15.43l.219-1.097h.379l.02.738.339-.738h.359l-.219 1.097h-.219l.179-.858-.379.858h-.219l-.02-.858-.179.858h-.26z"/></g></g><g><path d="M3.154 12.877c-.06.14-.259.239-.459.239H.222c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.279-.239.459-.239h2.473c.199 0 .299.12.239.239l-.798 1.815z" fill="none"/><path fill="none" d="M-.017 10.823h4.048v2.313H-.017z"/></g><g><path d="M37.157 12.877c-.06.14-.259.239-.459.239h-2.473c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.259-.239.459-.239h2.473c.199 0 .299.12.239.239l-.798 1.815z" fill="none"/><path fill="none" d="M33.986 10.823h4.048v2.313h-4.048z"/></g><g><path d="M20.125 4.103c-.06.14-.259.239-.459.239h-2.453c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.259-.259.439-.259h2.473c.199 0 .299.12.239.239l-.798 1.835z" fill="none"/><path fill="none" d="M16.955 2.029h4.048v2.313h-4.048z"/></g><g><path d="M20.125 21.732c-.06.14-.259.239-.459.239h-2.453c-.199 0-.299-.12-.239-.239l.798-1.815c.06-.14.259-.239.459-.239h2.473c.199 0 .299.12.239.239l-.818 1.815z" fill="none"/><path fill="none" d="M16.955 19.658h4.048v2.313h-4.048z"/></g><path fill="none" d="M4.032 4.322h29.954v15.316H4.032z"/><path fill="none" d="M-.017 2.029h38.011v19.943H-.017z"/></g></svg>
```

Comment:
Can we remove these attributes `version="1.1" id="pi-Layer_1" x="0" y="0" xml:space="preserve"`? It doesn't look like we need it.

Reply:
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24"...`

removed as per your request